code-challenges.test.js

```diff
@@ -15,42 +17,147 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+// describe("arrayFibz", () => {
+//     it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+//         const fibLength1 = 6
+//         // // Expected output: [1, 1, 2, 3, 5, 8]
+//         const fibLength2 = 10
+//         // // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+//         expect(arrayFibz(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//         expect(arrayFibz(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+
+//     })
+// })
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+//* RAN TEST (RESULTS BELOW): *//
 
+// ReferenceError: arrayFibz is not defined 
 
 // b) Create the function that makes the test pass.
 
+//Pseudocode:
+// Declare a function called arrayOfFibz. 
+// Set the parameter as an array called arrayFibonnaci. 
+// Use for loop to that takes in a number (greater than 2)
+// Return array with Fib sequence
+
+
+// const arrayFibz = (arrayFibonnaci) => {
+//     for (i =0; i<arrayFibonnaci.length, i++) {
+//         if (i > 2) {
+//             arrayFibonnaci[i] = array [i - 1] + arrayFibonnaci [i -2]; 
+//         } else {
+//             arrayFibonnaci[i] = 1;
+//         }
+//         return arrayFibonnaci
+//     }
+    
+//     // arrayFibonnaci.push(arrayFibonnaci[i-2] + arrayFibonnaci[i-1])
+//     // return i 
+
+// }
+
+// ** NOTES TO INSTRUCTOR **: I was getting a SyntaxError on Line 46 and I'm not sure why, Jest suggested a semicolon but that did nothing and seemed odd but honestly I'm not even sure the code would have worked without the error...still processing the logic and basics to be frank. (this is why my code above is commented out.)
```

Comment:
The syntax error was indeed a missing semicolon in your for statement. There should be a semicolon after `i<arrayFibonnaci.length` instead of a comma.  Common typo.  You put a lot of effort into this challenge.  I encourage you to expand your pseudocode so that it includes an explanation of the logic you'll use to generate the fibonacci sequence.  This one was difficult. Great effort!